src/java/org/apache/nutch/metadata/Nutch.java

```diff
@@ -80,4 +80,11 @@
 	public static final String STAT_PROGRESS = "progress";
 	/**Used by Nutch REST service */
 	public static final String CRAWL_ID_KEY = "storage.crawl.id";
+	
+	public static final String ARG_SEEDDIR = "url_dir";
+	public static final String ARG_CRAWLDB = "crawldb";
```

Comment:
@sujen1412 any comments on augmenting trivial Javadoc?
It makes a huge difference when documented.